create_jenkins_job.py

```diff
@@ -162,11 +162,13 @@ def main(argv=None):
         # many, likely related to qemu). Also disable linter tests (because
         # this is already a very slow job and because we lint plenty on other
         # jobs). Also don't build packages in parallel because we've seen hung
-        # builds that seem to be caused by parallelism.
+        # builds that seem to be caused by parallelism. Also don't rerun tests,
+        # as we're currently not paying attention to test failures.
         if os_name == 'linux-aarch64':
             job_data['dont_notify_every_unstable_build'] = 'true'
             job_data['ament_test_args_default'] = '--retest-until-pass 10 --ctest-args -LE linter --'
             job_data['ament_build_args_default'] = ''
+            job_data['ament_test_args_default'] = ''
```

Comment:
Do we want to run tests at all if we dont check the results ? that would be another use case to add the "run tests" checkbox to jenkins and disable it by default

If you want to run them once anyway, maybe we should keep `--ctest-args -LE linter --` to avoid running the linters event on the first round of testing